src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.test.tsx

```diff
@@ -52,64 +57,66 @@ describe('<BrowserRegionEditor />', () => {
     toggleRegionEditorActive: jest.fn()
   };
 
-  let wrapper: any;
-
   beforeEach(() => {
-    wrapper = mount(<BrowserRegionEditor {...defaultProps} />);
+    jest.resetAllMocks();
   });
 
   describe('rendering', () => {
-    test('contains Select', () => {
-      expect(wrapper.find(Select).length).toBe(1);
+    it('contains Select', () => {
+      const { container } = render(<BrowserRegionEditor {...defaultProps} />);
+      expect(container.querySelector('.select')).toBeTruthy();
     });
 
-    test('contains two Input elements', () => {
-      expect(wrapper.find(Input).length).toBe(2);
+    it('contains two input elements', () => {
+      const { container } = render(<BrowserRegionEditor {...defaultProps} />);
+      expect(container.querySelectorAll('input').length).toBe(2);
     });
 
-    test('contains submit and close buttons', () => {
-      expect(wrapper.find('button[type="submit"]')).toHaveLength(1);
+    it('contains submit and close buttons', () => {
+      const { container } = render(<BrowserRegionEditor {...defaultProps} />);
+      expect(container.querySelector('button[type="submit"]')).toBeTruthy();
```

Comment:
We are not checking for the close button as mentioned in the description.

Reply:
Yeah, the BrowserRegionEditor and BrowserRegionField components are pretty broken now, despite the passing tests. Don't know if I want to change anything about it in this PR, or fix both these two components and their tests in a separate PR.

Reply:
A separate PR would be better I think.